TWLight/applications/templates/applications/application_list.html

```diff
@@ -147,7 +147,7 @@ <h1 class="my-3">{{ title }}</h1>
                   </a></li>
             {% endfor %}
             <li class="page-item disabled"><a class="twl-links page-link">...</a></li>
-            <li class="page-item disabled"><a class="twl-links page-link" href="?page={{ object_list.previous_page_number}}{% if filters %}{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}">
+            <li><a class="twl-links page-link" href="?page={{ object_list.previous_page_number}}{% if filters %}{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}">
```

Comment:
Please add the class `page-item` to the `li` tag.

Reply:
> Please add the class `page-item` to the `li` tag.

Ah sorry! My bad. I have added it back again.